extensions/wsc/src/test/resources/config/noParams.xml

```diff
@@ -0,0 +1,14 @@
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:ws="http://www.mulesoft.org/schema/mule/wsc"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/wsc http://www.mulesoft.org/schema/mule/wsc/current/mule-wsc.xsd">
+
+    <ws:config name="config">
```

Comment:
`xmlns:ws="http://www.mulesoft.org/schema/mule/wsc"` it looks like the bug regarding `xmlns` was fixed =p